observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -6,6 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
```

Comment:
The note breaks the following guideline in the [RH SSG](https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc):
 
![Screenshot from 2024-07-30 17-13-43](https://github.com/user-attachments/assets/c73aabd9-7915-4fc7-8b14-e3540b88dd13)

Reply:
Maybe the note could be moved under the first paragraph in the Red Hat build of OpenTelemetry overview module.

Reply:
@dfitzmau, thank you. :sweat_smile: Legally speaking, I'm good because this snippet has been included in assemblies rather than modules.

In any case, I've moved the admonition include statement down the assembly files.